tools/issue-labeler/src/IssueLabelerService/Program.cs

```diff
@@ -7,34 +7,58 @@
 using AzureRagService;
 using Hubbup.MikLabelModel;
 using Azure.Identity;
-using Azure.Search.Documents;
 using System;
 using Azure.AI.OpenAI;
 using OpenAI.Chat;
 using Azure.Search.Documents.Indexes;
+using Microsoft.Extensions.Configuration;
+using IssueLabelerService;
 
 var host = new HostBuilder()
     .ConfigureFunctionsWebApplication()
     .ConfigureServices((context, services) => {
+        var functionConfig = context.Configuration;
+        var configEndpoint = new Uri(functionConfig["ConfigurationEndpoint"]);
+        var credential = new DefaultAzureCredential();
```

Comment:
This, we'll want to drive off of config.    DAC for local/dev loop and some specific one for production.   The Identity team is strongly pushing people to get off DAC in production scenarios.